classes/widgets.php

```diff
@@ -179,7 +179,7 @@ public function widget( $args, $instance ) {
 
 		$location_search .= apply_filters( 'sm-location-search-widget-before-submit', '' );
 
-		$location_search .= '<tr><td class="location_search_widget_submit_cell location_search_widget_cell"> <input type="submit" value="' . apply_filters( 'sm-search-label-search', __( 'Search', 'simplemap' ) ) . '" id="location_search_widget_submit_field" class="submit" /></td></tr>';
+		$location_search .= '<tr><td class="location_search_widget_submit_cell location_search_widget_cell"> <input type="submit" value="' . apply_filters( 'sm-search-label-search', __( 'Search', 'SimpleMap' ) ) . '" id="location_search_widget_submit_field" class="submit" /></td></tr>';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 287 characters

<!-- Comment by [SideCI](https://sideci.com) -->